core/src/main/resources/lib/form/repeatable/repeatable.js

```diff
@@ -163,27 +163,29 @@ Behaviour.specify("DIV.repeated-container", 'repeatable', -100, function(e) {
 });
 
     // button to add a new repeatable block
-Behaviour.specify("INPUT.repeatable-add", 'repeatable', 0, function(e) {
-        makeButton(e,function(e) {
+Behaviour.specify("BUTTON.repeatable-add", 'repeatable', 0, function(e) {
+        e.addEventListener('click', function(e) {
             repeatableSupport.onAdd(e.target);
         });
         e = null; // avoid memory leak
     });
+Behaviour.specify("INPUT.repeatable-add", 'repeatable', 0, function(e) {
```

Comment:
we can just add comma's between selectors apparently rather than duplicating: https://github.com/jenkinsci/jenkins/blob/master/war/src/main/webapp/scripts/hudson-behavior.js#L1213